radio/src/targets/horus/hal.h

```diff
@@ -727,6 +727,16 @@
   #define I2C_B1_GPIO_AF                  GPIO_AF_I2C3
   #define I2C_B1_SCL_GPIO_PinSource       GPIO_PinSource7
   #define I2C_B1_SDA_GPIO_PinSource       GPIO_PinSource8
+#elif defined(RADIO_FAMILY_T16)
```

Comment:
Please change to T16 only, as presently this would also affect e.g. TX16S and other T16 FAMILY radios (other than T18).

Reply:
Or even simpler, use on line 720:
`#if defined(RADIO_T18) || defined(RADIO_T16)`
then no duplication of defines will be required.

Reply:
thanks,  cleared and corrected